README.md

```diff
@@ -42,10 +49,10 @@ $ node example/parse.js -x 3 -y 4 -n5 -abc --beep=boop --no-ding foo bar baz
 # methods
 
 ``` js
-var parseArgs = require('minimist')
+const parseArgs = require('minimist');
 ```
 
-## var argv = parseArgs(args, opts={})
+## const argv = parseArgs(args, opts={})
```

Comment:
https://github.com/minimistjs/minimist#var-argv--parseargsargs-opts needs to keep working forever, so:
```suggestion
<a name="var-argv--parseargsargs-opts"></a>
## const argv = parseArgs(args, opts={})
```